src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs

```diff
@@ -70,18 +78,25 @@ public override void OnEnable()
         public override void OnDisable()
         {
             base.OnDisable();
+            gitPathView.OnDisable();
             DetachHandlers(Repository);
         }
 
         public override void OnDataUpdate()
         {
             base.OnDataUpdate();
+
+            if (gitPathView != null)
```

Comment:
Is this ever null here? We're not doing this check in any other method so it feels like it never is.

Reply:
👍